src/io/github/kounoike/Main.java

```diff
@@ -4,5 +4,6 @@
 
     public static void main(String[] args) {
 	// write your code here
+        System.out.println("Hello");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)